e2e/mock-api/src/app/responses.js

```diff
@@ -63,6 +63,20 @@ export const createStepUpUrl = (url) => {
   return redirectUrl.toString();
 };
 
+export const createStepUpHeader = (url) => {
+  // Grab the client's desired AM URL
+  const referer = new URL(url);
+  const amUrl = referer.searchParams.get('amUrl') || AM_URL;
+
+  // Base 64 of {"TransactionConditionAdvice":["39dfdd15-59a3-473c-a7fc-ecda3bbc3bc8"]}
+  const advices =
+    'eyJUcmFuc2FjdGlvbkNvbmRpdGlvbkFkdmljZSI6WyIzOWRmZGQxNS01OWEzLTQ3M2MtYTdmYy1lY2RhM2JiYzNiYzgiXX0=';
+  const format = '1';
```

Comment:
So far in my live testing I have only seen "format=1" - any idea what this means?

Reply:
I think `format=1` is the only format relevant to us. If I remember correctly, the other formats are for contexts the SDK doesn't handle.

Reply:
Though, explicitly asking this in the channel to which I just added you might be a good idea.

Reply:
The IG team have replied to say where this came from - looks like it was something you suggested in your gist before the PoC: https://gist.github.com/cerebrl/30e0da4b21a8b7b0eb6e2eed94279a7b